### ISSUE TITLE
Avoid the "Take-Home-Box" Overload

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#922]
<div dir="ltr" style="text-align: left;">
<i><b><span style="font-size: x-large;">Avoid the "Take-Home-Box" Overload</span></b></i><br>
<br><ins class="adsbygoogle" data-ad-client="ca-pub-2862230082201059" data-ad-slot="5129813120" style="display: inline-block; height: 280px; width: 336px;"></ins>
//<![CDATA[
(adsbygoogle = window.adsbygoogle || []).push({});
//]]>
<br>
After you&rsquo;ve had the RNY surgery, there is one thing you immediately begin to notice.. and that&rsquo;s what I call the &ldquo;Take-Home Box Overload&rdquo;.&nbsp; This is what happens inside your refrigerator as a result of you brining home all of those take-home boxes from restaurants because you cannot finish the meal.&nbsp; Even prior to surgery, you couldn&rsquo;t often finish the meal when some restaurants serve such large portions.&nbsp; So packing up the leftovers after an eat-out meal seems natural to you, so you do it often.<br><br>
PROBLEM ONE:&nbsp; The Take-Home Boxes can actually establish huge neighborhoods inside your fridge!&nbsp;&nbsp;&nbsp;<br><br>
PROBLEM TWO:&nbsp; You feel you MUST NOT throw all of that food out because you have been taught from the early days of your crib that you SHALL NOT WASTE FOOD because of all the starving children elsewhere who certainly would appreciate the food you are wasting, right?&nbsp; Frankly, when you heard that guilt-phrase that was supposed to motivate you to gobble up, didn&rsquo;t you feel even MORE compelled NOT to finish eating and MAIL your uneaten portion to those starving children?&nbsp; I never really understood that philosophy of telling me I better eat because some poor children don&rsquo;t have enough to eat&hellip;. A puzzlement.<br><br>
PROBLEM THREE:&nbsp; Because of the guilt, you decide that you&rsquo;ll just &ldquo;snack&rdquo; on the food in the take-home boxes.&nbsp; So you snack, snack again, snack some more, and HOORAY&hellip; you&rsquo;ve eaten everything in the box and you didn&rsquo;t waste the food!&nbsp; OH, but you&rsquo;ve done yourself a BAD turn!<br><br>
Bad turn?&nbsp; How?<br><br>
Let&rsquo;s consider that the food in that take-home box could contain 1500 to 2000 calories.&nbsp; So what?&nbsp; Well, when you eat those leftovers, you are consuming those ADDITIONAL 1500 to 2000 calories that you DON&rsquo;T need in your eating routine&hellip; especially if you eat them as SNACKS and not meals.&nbsp; It is a DANGEROUS thing to leave those take-home boxes in the fridge when they scream at you to be consumed.&nbsp; Don&rsquo;t listen to them!<br><br>
Here&rsquo;s what you need to do instead:<br><br>
If you truly plan on eating the leftovers as &ldquo;another meal portion&rdquo;, then you should wrap them according and place them in your FREEZER for eating LATER.&nbsp; You know that adage, &ldquo;out of sight, out of mind?&rdquo;&hellip; well, this is that basic concept.&nbsp; By doing this regularly, you&rsquo;ll be able to rotate your frozen meal portions for a better variety rather than eating the leftovers time after time.&nbsp; How appealing is it to eat six servings of chili in a row?&nbsp; It&rsquo;s not&hellip; so freeze the portions and eat them later.<br><br>
The FREEZER is your solution to curb the &ldquo;snacking&rdquo; out of those take-home boxes.&nbsp; So use it as your new best friend, you new best meal-planner buddy, and your calorie-saver!<br><br>
Getting use to the idea of tossing out food is a hard one, but it&rsquo;s something you have to get used to as well.&nbsp; The dog or cat will always be there to volunteer to eat what you can&rsquo;t&rsquo;, and the animals will show you a deeper devotion that ever imaginable!&nbsp; And your household loved ones often appreciate you giving them an open invitation to anything they find in the take-home boxes.&nbsp;<br><br>
Get a grip on this early so you won&rsquo;t sabotage yourself with those additional calories that will creep up on you if you fall pray to snacking or through guilt-eating.&nbsp; As an RNY patient, you are only to consume a minimal amount of calories anyway.. so don&rsquo;t fall into the trap of eating more than you need.&nbsp; This could be a factor as to why you seem to not be losing enough weight or why you are gaining some back after awhile.<br><br>
Bottom Line:&nbsp; Take-Home Boxes can be Dangerous&hellip;&nbsp; Tame them before they do you harm!<br><br><br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1Q0bZqx<br>
January 14, 2016 at 03:34PM
